### ISSUE TITLE
Update composer.json to upgrade nesbot/carbon to V3

### ISSUE THREAD

**Issue by phpfui** [Webklex/php-imap#537]
Happy to submit a PR for this, but I want to make sure it will be accepted first and that the project is still maintained.

Also it appears that https://github.com/Webklex/php-imap/pull/536 may fix PHP 8.4 nullable parameter depreciation.  I would like that as well.  Happy to submit a PR for that specific change if wanted.

**Comment by larsbo** [comment]
A PR is not needed, the composer.json has already been updated 8 months ago: https://github.com/Webklex/php-imap/commit/8696b28ba0899185f1250b54d10254f54afcb5bb 
But the release for this change is still missing.
@Webklex can you publish the change as a new release, please?

**Comment by it-can** [comment]
@Webklex can you release a new tag?

**Comment by mrhewitt** [comment]
Just in case it helps anybody, it appears this (upgrade nesbot/carbon to V3) may also fix issues you may encounter using this library with Symfony 7.1 

Tagged version 5.5 using old Carbon seemed to required symfony/translate for version 6, and refused to work in my 7.1 application, switching from 5.5 to dev-master "fixed" this (I know its not a solution really to use master branch and not a release, but just adding this in case anyone else battles with Symfony 7)

I do hope this is not abandoned, not sure how many others like myself are here because of PHP imap module being removed and depreciated from 8.4, would not bode well it this project was dead as well.

**Comment by phpfui** [comment]
I actually switched to javanile/php-imap2 which is more of a drop in replacement for the imap_ functions (just make them imap2_), but don't have it working just yet with my code, as I have not had enough time to debug it. Still trying to get other packages to support 8.4.

But yes, I think this PR should work and be approved. Shame to see a good package abandoned due to inactivity of the maintainer.

**Comment by larsbo** [comment]
Yes, maybe he can add more maintainer to the project to assist with updates.

@Webklex what do you think? I would like to help out. PS: I even live nearby you 😃
